app/Http/Controllers/DashboardController.php

```diff
@@ -1,19 +1,19 @@
 <?php
 
 namespace App\Http\Controllers;
-
 use Illuminate\Http\Request;
 
 class DashboardController extends Controller
 {
 
     /**
-     * Show the application home page.
+     * Show the application dashboard page.
      *
      * @return \Illuminate\Contracts\Support\Renderable
      */
-    public function index()
+    public function index(Request $request)
```

Comment:
how this is even working? if your route is a GET type 🤔  you are not getting any post/request data